_summaries/lang2018pointpillars_2.md

```diff
@@ -1,19 +1,51 @@
 ---
 layout: summary
 title: Summary
-paper: {{paper_tag}}
+paper: lang2018pointpillars
 # Please fill out info below
-author: # Your GitHub id
+author: aabayomi
 score: # How did you like this paper 0(dislike) to 10(love)
 ---
-
+<!-- 
 TODO: Summarize the paper:
 * What is the core idea?
 * How is it realized (technically)?
 * How well does the paper perform?
-* What interesting variants are explored?
+* What interesting variants are explored? -->
+
+# Core Idea
+
+The authors proposes as much faster approach by which encoder that learns point cloud representations by stacking learned features as vertical tensors. 
+
+# Main Contributions
+
+![](lang2018pointpillars_2a.png)
+
+* Representation of the 3-D lidar point cloud inputs as pseudo-images
+
+* 2-D convolutional backbone processes the images
+
+* Single-shot detectors regresses to 3D boxes
+
+# Loss function 
+
+The loss formulation combines the sotfmax classification loss, focal loss on object classification  and localization loss.This is optimized using Adam of learning rate of $2 * 10^−4$ with a decay rate 0.8 at every 15 epoch.
+
+![](lang2018pointpillars_2b.png)
+
+
+# Performance
+
+On KITTI datasets PointPillars  outperforms fusion based methods on cars and cyclists my respect to mean average precision (mAP) compared to lidar-only methods.
```

Comment:
typo my -> by

Reply:
You can add here that it performs better than SOTA both in terms of accuracy and speed. Maybe a line summarizing the ablation experiments will also be helpful.

Reply:
You can also add further to complete the summary about the interesting variants which were explored in the paper.